client/reader/following-manage/search-followed.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External Dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import SearchCard from 'components/search-card';
+
+class FollowingManageSearchFollowed extends Component {
+	static propTypes = {
+		value: PropTypes.string,
+		onSearch: PropTypes.func,
+	}
+
+	static defaultProps = {
+		onSearch: noop,
+	}
+
+	render() {
+		return (
+			<SearchCard
+				compact={ true }
+				isOpen={ true }
+				pinned={ false }
+				key="searchFollowed"
+				autoFocus={ false }
+				className="following-manage__search-followed"
+				additionalClasses="following-manage__search-followed-input"
+				placeholder={ this.props.translate( 'Search Followed Sites…' ) }
```

Comment:
I considered it, but I don't think the translated string is short enough to be visible as a placeholder in the input.